pages/contact-representative.js

```diff
@@ -21,17 +22,20 @@ const ContactRepresentative = (props) => {
         onHandleErrorClick={OnHandleErrorClick}
       />
       <Heading as="h1" size="LARGE">
-        Operator contact details
+        {props.t("Operator contact details")}
       </Heading>
       <Paragraph>
-        Please give us the details of the person at this company or charity we
-        should speak to about food hygiene and safety.
+        {props.t(
+          "Please give us the details of the person at this company or charity we should speak to about food hygiene and safety."
+        )}
       </Paragraph>
-      <HiddenTextAccessible summary={"What is a food business operator?"}>
+      <HiddenTextAccessible
+        summary={props.t("What is a food business operator?")}
```

Comment:
I think HiddenTextAccessible DOES need t={props.t} passed in as 't' is used within the component and it doesn't have all the props spread on. :)

Reply:
I've checked and tested this and it works fine, they actually don't need it passed as the translations are done on the actual component

Reply:
Yes, you're right, HiddenTextAccessible has the withTranslation wrapper so has it's own props.t, so doesn't need it passed in.